### ISSUE TITLE
Extension.metadata is missing

### ISSUE THREAD

**Issue by marcinjahn** [gjsify/gnome-shell#2]
In the [gjs extension guide](https://gjs.guide/extensions/development/creating.html#a-working-extension), there is the following code:

```js
const Me = ExtensionUtils.getCurrentExtension();
// ...
let indicatorName = `${Me.metadata.name} Indicator`;
``` 

In the types from this repository, the result of `ExtensionUtils.getCurrentExtension()` is an object conforming to the `Extension` interface, which does not have a `metadata` property:

```ts
export interface Extension {
    type: ExtensionType;
    state: ExtensionState;
    path: string;
    error: string | null;
    hasPrefs: boolean;
    hasUpdate: boolean;
    canChange: boolean;
}
```

**Comment by JumpLink** [comment]
@marcinjahn thank you for reporting, I hope to have the time for this the next days

**Comment by JumpLink** [comment]
Fixed in the current main branch, will be published with the next release

**Comment by JumpLink** [comment]
@marcinjahn released with 3.0.0-beta.14
